shared/wallets/wallet-list/wallet-row/container.js

```diff
@@ -20,7 +20,7 @@ const mapStateToProps = (state: TypedState, ownProps: {accountID: AccountID}) =>
 
 const mapDispatchToProps = (dispatch: Dispatch) => ({
   _onSelectAccount: (accountID: AccountID) => {
-    dispatch(createSelectAccount({accountID}))
+    dispatch(createSelectAccount({accountID, show: true}))
```

Comment:
This sets you to the account page of an account if you're on a different page, such as the settings page.